Task04.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+
+int GCD(int alpha, int beta) {
+    if(alpha == beta) return alpha;
+    int first = (alpha > beta) ? alpha : beta;
+    int second = (alpha < beta) ? alpha : beta;
+        do{
+             int remainder = first % second;
+             first = second;
+             second = remainder;
+        } while (first != 0);
+        return second;
+}
+
+void simplifyFraction(int& numerator, int& denominator) {
+    if(numerator == denominator) {
+        numerator = 1;
+        denominator = 1;
+    } else {
+    int commonDivisor = GCD(numerator, denominator); 
+    while(commonDivisor != 1) {
+        numerator /= commonDivisor;
+        denominator /= commonDivisor;
+        commonDivisor = GCD(numerator, denominator);
+    }
```

Comment:
This is gcd, you should just do, there won't be any other common divisors different than zero :)
        numerator /= commonDivisor;
        denominator /= commonDivisor;